tasks/deploy.yaml

```diff
@@ -26,8 +26,12 @@ tasks:
 
   - name: latest-package-release
     actions:
+      - description: "Get latest tag version from OCI"
```

Comment:
can you explain this change, seems unrelated to flavors?

Reply:
sure. The upgrade process got merged into main prior to me being able to get this merged. latest-package-release is a part of that change, not flavors.

Reply:
Full context on this one - https://github.com/defenseunicorns/uds-core/pull/113 - the previous logic was problematic in release please PRs.